modules/ingest-geoip/src/yamlRestTest/java/org/elasticsearch/ingest/geoip/IngestGeoIpClientYamlTestSuiteIT.java

```diff
@@ -31,6 +38,25 @@
 
 public class IngestGeoIpClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
 
+    private static final boolean useFixture = Booleans.parseBoolean(System.getProperty("geoip_use_service", "false")) == false;
```

Comment:
Why does this use `Booleans.parseBoolean`, but the other test FullClusterRestartIT uses `Boolean.getBoolean`?

Reply:
It sounds like we aren't applying the same forbidden apis rules to all source sets the same then. I only got a failure in this case.